src/helpers/networks.json

```
@@ -20,7 +20,8 @@
     "chainId": 7,
     "network": "mainnet",
     "rpcUrl": "https://rpc.dome.cloud",
-    "explorer": ""
+    "wsUrl": "wss://ws.dome.cloud",
+    "explorer": "http://exp.tch.in.th"
```

Can it be http**s**?
```suggestion
    "explorer": "https://exp.tch.in.th"
```

---

yes it can be https -> https://exp.tch.in.th
my discord username: s111s, #3726
telegram username: @samostzz

thanks